v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/CassandraAllDataTypesIT.java

```diff
@@ -94,6 +183,48 @@ public void allTypesTest() throws Exception {
     PipelineOperator.Result result =
         pipelineOperator().waitUntilDone(createConfig(jobInfo, Duration.ofMinutes(35L)));
     assertThatResult(result).isLaunchFinished();
+    LOG.info(
+        "Listing DLQ Entries from output director {} , dlq path {}, bucket {} ",
+        jobInfo.parameters().get("outputDirectory"),
+        getDlqPath(jobInfo),
+        getDlqBucketName(jobInfo));
+    List<String> dlqFiles = getDlqFileList(jobInfo);
+    LOG.info("DLQ Entries from {} are {}", getDlqPath(jobInfo), dlqFiles);
+    assertThat(dlqFiles).isNotEmpty();
+    // DLQ Replay.
+    FlexTemplateDataflowJobResourceManager dlqResourceManager = launchDlqReplay(jobInfo);
+    LaunchInfo dlqJobInfo = dlqResourceManager.getJobInfo();
+    Result dlqResult =
+        pipelineOperator()
+            .waitForCondition(
+                createConfig(dlqJobInfo, Duration.ofMinutes(10)),
```

Comment:
the default `DlqRetryMinutes` is set to 10 in live template, in that case any transient error will not be retried even once , maybe set retry minutes to 1 in the job config for better results